include/fmt/compile.h

```diff
@@ -37,6 +37,28 @@ struct is_compiled_string : std::is_base_of<compiled_string, S> {};
  */
 #define FMT_COMPILE(s) FMT_STRING_IMPL(s, fmt::detail::compiled_string)
 
+#if defined(__cpp_nontype_template_parameter_class) && \
+    (!FMT_GCC_VERSION || FMT_GCC_VERSION >= 903)
+template <typename Char, std::size_t N> struct fixed_string {
+  constexpr fixed_string(const Char (&str)[N + 1]) {
+    copy_str<Char, const Char*, Char*>(static_cast<const Char*>(str), str + N,
+                                       data);
+  }
+  Char data[N]{};
```

Comment:
`{}` seems unnecessary, please remove.

Reply:
It's either `{}` here or `: data{}` initialization in constructor for GCC 9.3, otherwise:
```
<source>:8:13: error: member 'fmt::v7::detail::fixed_string<char, 2>::data' must be initialized by mem-initializer in 'constexpr' constructor
<source>:12:8: note: declared here
   12 |   Char data[N];
      |        ^~~~
```